app/services/call_tool/call_status.rb

```diff
@@ -0,0 +1,18 @@
+module CallTool
+  module CallStatus
+    TIME_OUT_THRESHOLD = 15.seconds
+
+    # Returns one of: connecting, ringing, answered, completed, timed_out
+    def self.for(call)
+      status = call.member_call_events.last&.dig('CallStatus')
```

Comment:
But if there's no `last`, can you use the safe navigation operator? `.last&['CallStatus']`